composer.json

```diff
@@ -0,0 +1,45 @@
+{
+  "name": "zfr/zfr-oauth2-server",
+  "description": "PHP library to create an OAuth 2 server",
+  "type": "library",
+  "license": "MIT",
+  "keywords": [
+    "oauth",
+    "oauth 2",
+    "server"
+  ],
+  "homepage": "http://www.github.com/zf-fr/zfr-oauth2-server",
+  "authors": [
+    {
+      "name": "Michaël Gallego",
+      "email": "mic.gallego@gmail.com",
+      "homepage": "http://www.michaelgallego.fr"
+    }
+  ],
+  "require": {
+    "php": ">=7.0",
+    "doctrine/common": "~2.5",
+    "zendframework/zend-diactoros": "~1.1",
+    "container-interop/container-interop": "^1.1"
+  },
+  "require-dev": {
+    "phpunit/phpunit": "~5.1",
+    "squizlabs/php_codesniffer": "^2.5",
+    "satooshi/php-coveralls": "~0.6",
+    "doctrine/orm": "~2.5",
```

Comment:
I'd like to remove this if possible. It should work with any Doctrine libraries, not just only ORM (should work with Mongo...).